### ISSUE TITLE
[SF][0.131.24][DynamicPage][a11y] Contentinfo landmark should not be contained in another landmark

### ISSUE THREAD

**Issue by i323808** [SAP/ui5-webcomponents-react#2421]
**Describe the bug**
Contentinfo landmark should not be contained in another landmark.

**Isolated Example**
https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-dynamicpage--with-floating-footer

**To Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-dynamicpage--with-floating-footer
2. Click on 'axe Dev Tools'

**Expected behavior**
Contentinfo landmark should not be contained in another landmark.

**Screenshots**
![a11y_contentinfo1](https://user-images.githubusercontent.com/30744017/146322564-e51213d7-0883-4ed4-8e28-69efe3fb175a.png)

![a11y_contentinfo2](https://user-images.githubusercontent.com/30744017/146322585-ba92e46e-47f8-4ec2-a433-02e4b26e0a2a.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 0.31.25
`@ui5/webcomponents-react` version: 0.131.24
Operating System: macOS Monterey
Browser: Chrome Version 96.0.4664.93 (Official Build) (x86_64)

**Comment by GongRichard** [comment]
Hi @Lukas742 ,
Can this issue be fixed before Apr 1st?
Thanks,
Richard

**Comment by Lukas742** [comment]
Hi @GongRichard 

since we will first need a concept for this, I can't give you an estimate when this will be finished. I'll try to look into it next week.

**Comment by Lukas742** [comment]
Hi @GongRichard 

after some discussions we decided to let the app developer decide which role should be applied to the `footer` of the DynamicPage. When the linked PR is merged and released, it'll be possible to set the role of the footer by using the `a11yConfig` prop. E.g.:  `a11yConfig={{ dynamicPageFooter: { role: 'toolbar' } }}`

For the FlexibleColumnLayout you can use the `accessibilityRoles` dom ref property ([see here](https://github.com/SAP/ui5-webcomponents-react/blob/main/packages/main/src/webComponents/FlexibleColumnLayout/index.tsx#L61-L69)) to change the role of the respective column. 
E.g. for initial role change:
```
...
const fclRef = useRef();
useEffect(() => {
  customElements.whenDefined('ui5-flexible-column-layout').then(() => {
    if (fclRef.current) {
      fclRef.current.accessibilityRoles = { startColumnRole: "region", midColumnRole: "region" };
    }
  });
}, []);
```
